tests/unit/test_rule_tester/test_rule_tester.py

```diff
@@ -206,7 +206,7 @@ def test_rule_with_define_xml(define_xml_variable_validation_rule: dict):
         }
     ]
 
-    with open(test_define_file_path, "rb") as file:
+    with open(test_define_file_path, "r") as file:
```

Comment:
Why change this if we are expecting the file in bytes

Reply:
expecting in str, so i updated the type